server/liveblog/client_modules/client_modules.py

```diff
@@ -160,8 +174,8 @@ class ClientBlogPostsService(BlogPostsService):
     def get(self, req, lookup):
         cache_key = 'lb_ClientBlogPostsService_get_%s' % (hash(frozenset(req.__dict__.items())))
         blog_id = lookup.get('blog_id')
-        docs = app.blog_cache.get(blog_id, cache_key)
+        docs = app.liveblog_cache.get(blog_id, cache_key)
```

Comment:
5 lines here are the same as 5 lines above. Please put in a separate method. Also, is it truly impossible in the code to update the document without updating the cache?